operator/config/manager/kustomization.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
-  newTag: latest
+  newName: quay.io/konflux-workspaces/workspaces-operator
+  newTag: v0.1.0-alpha2
```

Comment:
should we revert this?

Reply:
Nope, we need it this time.  kube-linter denies any image tagged with `:latest`.

Reply:
Fixed in 21d3b5e.